### PR TITLE
Lesson 14

### PR DESCRIPTION
**Pull request by impactmass**

### CODE REVIEW

File: lesson14/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## 1. Introduction to Flexbox
+
+Flexbox is a comparatively newer way to layout elements on the page without the limits and frustration of Floats like:
+  - You have to clear Floats (when the containing element collapses).
+  - Sometimes have to change the order of your HTML to achieve a Floats design
+
+Flexbox layout makes it easier to design flexible responsive layout structure (hint: Flexbox = flexible boxes).
+
+With flexbox, you can align elements horizontally and vertically. Unlike floats, you can re-order flex items without needing to change the HTML
```
Comment (angelsvirkov) [inline lesson14/index.md:19]:
```suggestion
With flexbox, you can align elements horizontally and vertically. Unlike floats, you can re-order flex items without needing to change the HTML.
```

File: lesson14/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+## Class Agenda
+
+1. Introduction to Flexbox
+1. The Flex model
+
+## 1. Introduction to Flexbox
+
+Flexbox is a comparatively newer way to layout elements on the page without the limits and frustration of Floats like:
+  - You have to clear Floats (when the containing element collapses).
+  - Sometimes have to change the order of your HTML to achieve a Floats design
```
Comment (angelsvirkov) [inline lesson14/index.md:15]:
```suggestion
  - Sometimes have to change the order of your HTML to achieve a Floats design.
```

File: lesson14/index.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+Flexbox layout makes it easier to design flexible responsive layout structure (hint: Flexbox = flexible boxes).
+
+With flexbox, you can align elements horizontally and vertically. Unlike floats, you can re-order flex items without needing to change the HTML
+
+## 2. The Flex Model
+
+- `flex` is the value for the display property that activates the Flexbox layout on an element.
+- What is a flex container?
+- What are flex items?
+- Explain some properties that can be used on a flex container
```
Comment (angelsvirkov) [inline lesson14/index.md:26]: I find it a little bit non-fluent that we have a set of questions and somewhere in between the imperative sentence `Explain ....` 

```suggestion
- What are the most common properties that can be used on a flex container?
```

File: lesson14/index.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  - `flex-direction` (default `row`)
+  - `flex-wrap` (default `nowrap`)
+  - `justify-content` (default: `flex-start`)
+  - `align-items` (default: `stretch`)
+  - `align-content` (default: `stretch`)
+  - `flex-flow`
+- Explain some properties that can be used on a flex item
```
Comment (angelsvirkov) [inline lesson14/index.md:33]:
```suggestion
- What are the most common properties that can be used on a flex item?
```

File: lesson14/index.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  - `align-self`, similar to `align-items` on the container, but applies to a single flex item.
```
Comment (angelsvirkov) [inline lesson14/index.md:34]:
```suggestion
  - `align-self` is similar to `align-items` on the container, but applies to a single flex item.
```

File: lesson14/index.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  - `flex`: shorthand property for the `flex-grow`, `flex-shrink`, and `flex-basis`
+  - `flex-grow`: specifies how an item grows when they're allowed to take extra space available in the container (default: 0)
```
Comment (angelsvirkov) [inline lesson14/index.md:36]:
```suggestion
  - `flex-grow`: specifies how an item grows when it is allowed to take extra space available in the container (default: 0)
```

File: lesson14/index.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+  - `flex-shrink` specifies how an item shrinks when there is not enough space available in the container (default: 1)
+  - flex-basis
+  - order
+
+
+# Exercise Description
+1. Use Flexbox instead of float to style the navigation bar on the portfolio in [this CodeSandbox](https://codesandbox.io/s/flex1-vkyi2)
+2. Use Flexbox to style the HTML form provided in [this CodeSandbox](https://codesandbox.io/s/flexbox2-ymdfj?file=/index.html) (see attached image for expected result)
```
Comment (angelsvirkov) [inline lesson14/index.md:44]: Wooow good job here 💯  🚀 . Maybe we can use similar exercises for our exercise session (after making sure everyone catches up with the latest checkpoint)

File: lesson14/index.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+3. Style the Flexbox Pricing Grid in [this CodeSandbox](https://codesandbox.io/s/flexbox3-w8vqk) (see attached image for expected look)
+
+# Homework
+
+- Checkpoint 14
```
Comment (angelsvirkov) [inline lesson14/index.md:49]:
```suggestion
- Bring your website to Checkpoint 14.
```

File: lesson14/index.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+# Resources
```
Comment (angelsvirkov) [inline lesson14/index.md:51]:
```suggestion
# Glossary

- Flexbox - Flexible box

# Resources
```

**Review by angelsvirkov — CHANGES_REQUESTED** [review]
Looks very good. Good job. Here are some typos and some improvement suggestions

File: lesson14/index.md (hunk 3 above)
Comment (impactmass) [reply lesson14/index.md:26]: yes, good point. Thanks

File: lesson14/index.md (hunk 7 above)
Comment (impactmass) [reply lesson14/index.md:44]: Yes. I had that idea too when doing this 😊

**Review by angelsvirkov — APPROVED** [review]
@impactmass you could go ahead and merge 💯

**Pull request merged**